.travis.yml

```diff
@@ -29,7 +29,12 @@ before_install:
   - sudo locale-gen fr_FR.UTF-8 tr_TR.UTF-8
 install:
   - pip install tox==2.5.0
-script: tox -e $TOX_ENV
+script:
+  - tox -e $TOX_ENV
+  # testing installing with pip
+  - pip install -e .
+  - python -c "import pelican; import pelican.tools; import pelican.tests"
```

Comment:
This will always succeed when you do `-e` install.

Reply:
Can you install the current directory using pip without `-e`? If `-e` is `setup.py develop`, is there an equivalent to `setup.py install`?

Reply:
Sure, `pip install .`. But, honestly I'd prefer a new environment without [`usedevelop`](https://github.com/getpelican/pelican/blob/master/tox.ini#L11) and actually running the unittests:
```
python -Wd unittest discover pelican.tests
```

Reply:
I'm not familiar with tox or what usedevelop is. Or what that python command does. Maybe you can recommend the complete changes you'd like to see in this PR? It will take me some time to learn what it all is/does.